packages/daimo-userop/src/daimoOpBuilder.ts

```diff
@@ -62,7 +71,7 @@ export class DaimoOpBuilder extends UserOperationBuilder {
         Presets.Middleware.estimateUserOperationGas(instance.provider)
       )
       .useMiddleware(async (ctx) => {
-        ctx.op.verificationGasLimit = 500_000;
+        ctx.op.verificationGasLimit = DEFAULT_USEROP_VERIFICATION_GAS_LIMIT;
```

Comment:
is this change from 500k -> default intended?

Reply:
yes, didn't want to hardcode in `userop`.

currently,
- verification actually costs ~400k gas, most of that in P256Verifier
- verificationGasLimit 500k **should** work
- does work in actual execution
- fails with that enigmatic "too much token" error in estimateUserOperationGas


so for now `DEFAULT_USEROP_VERIFICATION_GAS_LIMIT` is 1.5m, but we can set it to 500k once that last issue is resolved in pimlico